AlicerceTests/ServiceLocatorTests.swift

```diff
@@ -0,0 +1,695 @@
+//
+//  ServiceLocatorTests.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 26/01/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+class ServiceLocatorTests : XCTestCase {
+    
+    var serviceLocator = ServiceLocator()
```

Comment:
Thats fair enough. I too normally hate implicitly unwrapped options, but sometimes make an exception for unit tests as they are not production code. But thats fair enough 👍